### ISSUE TITLE
SLF4J log error in stdout console

### ISSUE THREAD

**Issue by Shini31** [sscarduzio/elasticsearch-readonlyrest-plugin#255]
Hello,

With ROR 1.16.8 and 1.16.9-pre4, i have a SLF4J log error in stdout console for each elasticsearch restart (ES 5.5.0 and Ubuntu 14.04).
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```
How to fix that ?

Thanks.

**Comment by sscarduzio** [comment]
Thanks for reporting @Shini31, this is fixed missing dependency, will be on the next stable release.
If you notice anything else, let me know!

**Comment by Shini31** [comment]
Thanks for you work and your reactivity !

**Comment by chaimyom** [comment]
Hello,

I have the same  issue, on version 5.3.0, did you fix this on this version too? 

Thanks.

**Comment by r-m-n** [comment]
Hello,

The same issue with ROR 1.16.9 and ES 5.5.1

**Comment by sscarduzio** [comment]
Yes. Unfortunately I had to revert this because it was causing all kind of issues preventing ES to even start. Very sorry :( 

I spent half an hour on this, but still couldn't figure it out. Feel free to experiment with adding/removing jars in your ES installation to find a working combination! I'll come back to this in the future (as it doesn't really look as a blocker).

**Comment by abdox2** [comment]
Adding "slf4j-simple-1.7.25.jar" to the zip readonlyrest-plugin and reinstall solve the problem.

http://www.slf4j.org/download.html

**Comment by sscarduzio** [comment]
@abdox2 can you fix this at gradle level in ROR?

**Comment by abdox2** [comment]
it's been fixed c08cd639c9c651fd6b72b14e786a8c6e685b4eb9 and revert 0179bbdf6b6a2b17d5130d48c2507e7bd1e24cb3 for some reason. for me it's works just fine with the slf4j-simple.

Readonlyrest 1.16.13
Elasticsearch 5.6.4
Ubuntu 16.04.3

**Comment by sscarduzio** [comment]
guys, I think I solved this, can you also check? (1.16.15 is out now)

**Comment by kvaapagarrot** [comment]
hi, i have the same issue but with ROR 1.18.1_es6.8.0
i added into zip slf4j-simple-1.7.25.jar and reinstall plugin but it doesn't help anyway

**Comment by drdev** [comment]
Still happening on newest version available:

**Plugin information**:

```
Plugins directory: /usr/share/elasticsearch/plugins
readonlyrest
- Plugin information:
Name: readonlyrest
Description: Safely expose Elasticsearch REST API
Version: 1.19.5
Elasticsearch Version: 7.7.0
Java Version: 1.8
Native Controller: false
Extended Plugins: []
 * Classname: tech.beshu.ror.es.ReadonlyRestPlugin
```

**Logs messages**:

```
[2020-05-29T10:01:26,952][INFO ][o.e.n.Node               ] [server.local]started
[2020-05-29T10:01:44,948][INFO ][t.b.r.b.Ror$             ] [server.local]Loading ReadonlyREST settings from index failed: cannot find index
[2020-05-29T10:01:44,949][INFO ][t.b.r.b.Ror$             ] [server.local]Loading ReadonlyREST settings from file: /etc/elasticsearch/readonlyrest.yml
[2020-05-29T10:01:45,906][WARN ][stderr                   ] [server.local]SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[2020-05-29T10:01:45,906][WARN ][stderr                   ] [server.local]SLF4J: Defaulting to no-operation (NOP) logger implementation
[2020-05-29T10:01:45,906][WARN ][stderr                   ] [server.local]SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

**Comment by askids** [comment]
I encountered this same error today when running ES v7.8.1 with 1.24.0 of free ROR ES plugin.

**Comment by rapkaan** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 6, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
matthew-mcateer, @samcus, @yifuguo, @kevinschenkers, @Phantom0008, @kcyeu, @flyingfork21, @Edulongo7, @MoriZalev, @antonkovernik, @vmonaco, @vic0, @lns, @maxgfr, @mwishek, @j0eii, @dafinguzman, @jeremi, @spf13, @wescleveland

**Comment by rapkaan** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 6, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
Su-yj, @lunatique, @MemeSkywalker, @jrmeier, @andrewpaulicek, @andrei15, @bellyfat, @vishnubob, @northeastsquare, @dobestan, @fragosti, @nishino-tsukasa, @worldburger, @mattreiss, @SilaEsme, @nerdfiles, @gianlu91, @weinerk, @kii1998, @randomizer333
